helm/starboard-exporter/values.schema.json

```diff
@@ -12,7 +32,7 @@
                             "type": "boolean"
                         },
                         "targetLabels": {
-                            "type": ["null", "array"]
+                            "type": "null"
```

Comment:
This will still need to have "array" in there. Otherwise the chart will fail validation if a user supplies a list of labels

Reply:
I thought I added `array` field but actually I removed it. Reverted it back.